landing-page/content/common/how-to-release.md

```diff
@@ -263,64 +292,102 @@ Java artifacts are available from Maven Central.
 Thanks to everyone for contributing!
 ```
 
-### Documentation Release
+#### Codebase updates
+
+There are a few places in the codebase that reference the latest Iceberg release version number and need to be updated:
+1. https://github.com/apache/iceberg/blob/master/.github/ISSUE_TEMPLATE/iceberg_bug_report.yml
+2. https://github.com/apache/iceberg/blob/master/python/dev/Dockerfile
+3. `oldVersion` in `revapi` in https://github.com/apache/iceberg/blob/master/build.gradle
```

Comment:
Do we need to modify all these values in the release commit?

Reply:
What do you mean by release commit? 

I think this can only be changed after the artifact is released and fully published, so there is a period of time that we have released but still cannot consume the artifact.

Reply:
I guess I'm just confused about what these instructions are telling me to do, like after release am I making a new commit that updates these things?

Reply:
yes, usually we did those updates after a release (at least the revApi `oldVersion` bump you can only do after artifacts are available)